options/locale/locale_en-US.ini

```
@@ -2446,6 +2446,13 @@ settings.thread_id = Thread ID
 settings.matrix.homeserver_url = Homeserver URL
 settings.matrix.room_id = Room ID
 settings.matrix.message_type = Message Type
+settings.visibility.private.button = Make Private
+settings.visibility.private.text = Making private the repo will do that only allowed members can see.
```

To me this reads weird. Maybe we could reword this to the following:
```suggestion
settings.visibility.private.text = Changing the visibility to private will make the repo visible to only allowed members.
```